packages/init2/src/lib/repositoryDomain.ts

```diff
@@ -0,0 +1,121 @@
+const ABC123 = "abcdefghijklmnopqrstuvwxyz0123456789";
+
+// 11 characters long or less adjectives
+const ADJECTIVES = [
+	"adorable",
+	"beautiful",
+	"charming",
+	"cloudy",
+	"delightful",
+	"focused",
+	"honest",
+	"inspiring",
+	"lovable",
+	"melodious",
+	"pleasant",
+	"pretty",
+	"shiny",
+	"sugary",
+	"tall",
+	"valliant",
+].filter((adjective) => adjective.length <= 11);
+
+// 11 characters long or less bakeries
+const BAKERIES = [
+	"baguette",
+	"briouat",
+	"crepe",
+	"croissant",
+	"donut",
+	"eclair",
+	"flaons",
+	"kolompeh",
+	"macaron",
+	"paste",
+	"pretzel",
+	"rustico",
+	"semla",
+	"strudel",
+	"taiyaki",
+	"toast",
+].filter((bakery) => bakery.length <= 11);
```

Comment:
This is my second favorite part of this PR.

Reply:
I feel like this overall file could be a package of its own, Sindre's style